README.md

```diff
@@ -240,6 +259,48 @@ Use the following command to view the database within the MongoDB container.
 docker exec -it rosalution_rosalution-db_1 mongosh rosalution_db
 ```
 
+### Production
+
+#### Using the Build Script
+
+When deploying the Rosalution system in a production environment, it is important to ensure that the build is optimized
```

Comment:
"... it is important to ensure that the build is optimized for security when deploying the Rosalution system in a production environment. This is achieved with the `build.sh` script which ..."

Reply:
I second this suggestion. It gives more detail.

Reply:
I believe I have made this requested change